exonum/src/runtime/mod.rs

```diff
@@ -210,13 +211,13 @@ impl From<RuntimeIdentifier> for u32 {
 /// COMMIT ::= deploy_artifact* commit_service* after_commit
 /// ```
 ///
-/// The ordering for the "readonly" method `is_artifact_deployed` in relation
+/// The ordering for the "read-only" method `is_artifact_deployed` in relation
 /// to the lifecycle above is not specified.
 ///
 /// # Consensus and Local Methods
 ///
-/// The following methods should return the same result given the same arguments for all nodes
-/// in the blockchain network:
+/// The following methods should return the same result if they contain the same arguments for all
```

Comment:
This phrasing looks a bit weird: methods do not contain arguments, but arguments are provided into methods (to obtain some result).